packages/terra-table/tests/wdio/table-spec.js

```diff
@@ -157,17 +157,29 @@ Terra.describeViewports('Table', ['medium', 'large'], () => {
       Terra.validates.element('scrollable-table-second-focus', { selector: scrollableTableSelector });
     });
 
-    it('Validates a table has zebra striping when specified via props', () => {
+    it('Validates table is scrollable vertically via arrow keys', () => {
       browser.keys(['Tab', 'ArrowDown', 'ArrowDown']);
       Terra.validates.element('scrollable-table-first-scroll', { selector: scrollableTableSelector });
     });
 
-    it('Validates second scrollable table receives focus', () => {
+    it('Validates table is scrollable horizontally via arrow keys', () => {
       browser.keys(['Tab', 'Tab', 'ArrowRight', 'ArrowRight']);
       Terra.validates.element('scrollable-table-second-scroll', { selector: scrollableTableSelector });
     });
   });
 
+  describe('Table with Sections', () => {
```

Comment:
Should we also include tests for collapsible sections?

Reply:
The collapsible sections exist in the test file so they should be covered.